cas-management-webapp-support/src/main/java/org/apereo/cas/mgmt/config/CasManagementWebAppConfiguration.java

```diff
@@ -0,0 +1,346 @@
+import org.jasig.inspektr.audit.AuditTrailManagementAspect;
+import org.jasig.inspektr.audit.AuditTrailManager;
+import org.jasig.inspektr.audit.spi.AuditActionResolver;
+import org.jasig.inspektr.audit.spi.AuditResourceResolver;
+import org.jasig.inspektr.audit.spi.support.DefaultAuditActionResolver;
+import org.jasig.inspektr.audit.spi.support.ObjectCreationAuditActionResolver;
+import org.jasig.inspektr.audit.spi.support.ParametersAsStringResourceResolver;
+import org.jasig.inspektr.audit.support.Slf4jLoggingAuditTrailManager;
+import org.jasig.inspektr.common.spi.PrincipalResolver;
+import org.pac4j.cas.client.CasClient;
+import org.pac4j.core.authorization.AuthorizationGenerator;
+import org.pac4j.core.authorization.RequireAnyRoleAuthorizer;
+import org.pac4j.core.client.Client;
+import org.pac4j.core.config.Config;
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.profile.UserProfile;
+import org.pac4j.springframework.web.RequiresAuthenticationInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.springframework.core.io.Resource;
+import org.springframework.util.StringUtils;
+import org.springframework.web.filter.CharacterEncodingFilter;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+import org.springframework.web.servlet.handler.SimpleUrlHandlerMapping;
+import org.springframework.web.servlet.i18n.CookieLocaleResolver;
+import org.springframework.web.servlet.mvc.Controller;
+import org.springframework.web.servlet.mvc.ParameterizableViewController;
+import org.springframework.web.servlet.mvc.SimpleControllerHandlerAdapter;
+import org.springframework.web.servlet.mvc.UrlFilenameViewController;
+import org.springframework.web.servlet.view.RedirectView;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Properties;
+
+/**
+ * This is {@link CasManagementWebAppConfiguration}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+@Configuration("casManagementWebAppConfiguration")
+public class CasManagementWebAppConfiguration extends WebMvcConfigurerAdapter {
+
+    private static final String AUDIT_ACTION_SUFFIX_FAILED = "_FAILED";
+    private static final String AUDIT_ACTION_SUFFIX_SUCCESS = "_SUCCESS";
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['VARIABLE_DEF' should be separated from previous statement.](https://www.codacy.com/app/mmoayyed/cas/file/2523079437/issues/source?bid=3256115&fileBranchId=3280886#l63)